src/main/java/org/gridsuite/study/server/service/LoadFlowService.java

```diff
@@ -282,4 +230,123 @@ public List<LimitViolationInfos> getLimitViolations(UUID nodeUuid) {
         }
         return result;
     }
+
+    public LoadFlowParametersInfos getLoadFlowParameters(UUID parametersUuid) {
+        LoadFlowParametersInfos parameters;
+
+        String path = UriComponentsBuilder.fromPath(DELIMITER + LOADFLOW_API_VERSION + PARAMETERS_URI)
+            .buildAndExpand(parametersUuid).toUriString();
+        try {
+            parameters = restTemplate.getForObject(loadFlowServerBaseUri + path, LoadFlowParametersInfos.class);
+        } catch (HttpStatusCodeException e) {
+            if (HttpStatus.NOT_FOUND.equals(e.getStatusCode())) {
+                throw new StudyException(LOADFLOW_PARAMETERS_NOT_FOUND);
+            }
+            throw handleHttpError(e, GET_LOADFLOW_PARAMETERS_FAILED);
+        }
+        return parameters;
+    }
+
+    public UUID createLoadFlowParameters(String parameters) {
+
+        Objects.requireNonNull(parameters);
+
+        var path = UriComponentsBuilder
+                .fromPath(DELIMITER + LOADFLOW_API_VERSION + "/parameters")
+                .buildAndExpand()
+                .toUriString();
+
+        HttpHeaders headers = new HttpHeaders();
+        headers.setContentType(MediaType.APPLICATION_JSON);
+
+        HttpEntity<String> httpEntity = new HttpEntity<>(parameters, headers);
+
+        UUID parametersUuid;
+
+        try {
+            parametersUuid = restTemplate.exchange(loadFlowServerBaseUri + path, HttpMethod.POST, httpEntity, UUID.class).getBody();
+        } catch (HttpStatusCodeException e) {
+            throw handleHttpError(e, CREATE_LOADFLOW_PARAMETERS_FAILED);
+        }
+
+        return parametersUuid;
```

Comment:
You can return inside the try block to be more simple
```suggestion
        try {
            return restTemplate.exchange(loadFlowServerBaseUri + path, HttpMethod.POST, httpEntity, UUID.class).getBody();
        } catch (final HttpStatusCodeException ex) {
            throw handleHttpError(ex, CREATE_LOADFLOW_PARAMETERS_FAILED);
        }
```